exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,207 @@
+{
+  "exercise": "armstrong-number",
+  "version": "1.0.0",
+  "comments": [
+    "Both input-parameters are always given in base-10(decimal)."
+  ],
+  "cases": [
+    {
+      "description": "0 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 0,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "1 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 1,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "2 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 2,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "9 is an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 9,
+        "base": 10
+      },
+      "expected": true
+    },
+    {
+      "description": "10 is not an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 10,
+        "base": 10
+      },
+      "expected": false
+    },
+    {
+      "description": "100 is not an Armstrong number in base 10",
+      "property": "armstrong",
+      "input": {
+        "number": 100,
+        "base": 10
+      },
+      "expected": false
+    },
+    {
+      "description": "111 is not an Armstrong number in base 10",
```

Comment:
(Feedback applies to all tests.)

The description here doesn't tell me anything already covered by the input and output values.
What is this test testing?
Why is 111 useful number to test?
If we had a solution that passed the previous n tests, what code would need to change that would make this test pass?

Reply:
I just wanted to have another false 3-digit 10-base test additionally to 100.
I guess you are right and this does not catch anything special additionally.
=> Removed

Reply:
Why is 100 useful to test?